kernelci/rootfs.py

```diff
@@ -81,3 +93,86 @@ def upload(api, token, upload_path, input_dir):
             px = os.path.relpath(root, input_dir)
             artifacts[os.path.join(px, f)] = open(os.path.join(root, f), "rb")
     upload_files(api, token, upload_path, artifacts)
+
+
+def validate(configs):
+    err = sort_check(configs['rootfs_configs'])
+    if err:
+        print("Rootfs broken order: '{}' before '{}".format(*err))
+        return False
+    for name, config in configs['rootfs_configs'].items():
+        if config.rootfs_type == 'debos':
+            return _validate_debos(name, config)
+        elif config.rootfs_type == 'buildroot':
+            return _validate_buildroot(name, config)
+        else:
+            print('Invalid rootfs type {} for config name {}'
+                  .format(config.rootfs_type, name))
+            return False
+
+
+def _validate_debos(name, config):
+    err = sort_check(config.arch_list)
+    if err:
+        print("Arch order broken for {}: '{}' before '{}".format(
+            name, err[0], err[1]))
+        return False
+    err = sort_check(config.extra_packages)
+    if err:
+        print("Packages order broken for {}: '{}' before '{}".format(
+            name, err[0], err[1]))
+        return False
+    err = sort_check(config.extra_packages_remove)
+    if err:
+        print("Packages order broken for {}: '{}' before '{}".format(
+            name, err[0], err[1]))
+        return False
+    return True
+
+
+def _validate_buildroot(name, config):
+    err = sort_check(config.arch_list)
+    if err:
+        print("Arch order broken for {}: '{}' before '{}".format(
+            name, err[0], err[1]))
+        return False
+    err = sort_check(config.frags)
+    if err:
+        print("Frags order broken for {}: '{}' before '{}".format(
+            name, err[0], err[1]))
+        return False
+    return True
+
+
+def dump_configs(configs):
```

Comment:
Missing docstring.

Reply:
Same comment about `kernelci.config.rootfs` as a potentially better location for this code.

Reply:
Applied changes